sorts/shell_sort.py

```diff
@@ -1,13 +1,5 @@
 """
-This is a pure Python implementation of the shell sort algorithm
-
-For doctests run following command:
-python -m doctest -v shell_sort.py
-or
-python3 -m doctest -v shell_sort.py
-
-For manual testing run:
-python shell_sort.py
+https://en.wikipedia.org/wiki/Shellsort#Pseudocode
```

Comment:
I would leave the first line like this
```suggestion
"""
This is a Python implementation of the shell sort algorithm.

Reference: https://en.wikipedia.org/wiki/Shellsort#Pseudocode
```